weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/sections/ExampleCompareSection/exampleCompareSectionUtil.ts

```diff
@@ -118,19 +118,19 @@ const rowIsSelected = (
 export const useFilteredAggregateRows = (state: EvaluationComparisonState) => {
   const leafDims = useMemo(() => getOrderedCallIds(state), [state]);
   const compositeMetricsMap = useMemo(
-    () => buildCompositeMetricsMap(state.data, 'score'),
-    [state.data]
+    () => buildCompositeMetricsMap(state.summary, 'score'),
+    [state.summary]
   );
 
   const flattenedRows = useMemo(() => {
     const rows: FlattenedRow[] = [];
-    Object.entries(state.data.resultRows).forEach(
+    Object.entries(state.results.result?.resultRows ?? {}).forEach(
```

Comment:
this is kinda weird, this used to be a real data object but now its a query response?

Reply:
like, why are we storing `results.result` instead of `result`

Reply:
i modified the name to be more clear